README.md

```diff
@@ -37,7 +37,7 @@ This project provides Kubernetes administrators with the topology data related t
 | Storage System Volume Name | The name of the volume on the storage system that is associated with the persistent volume                                                         |
 | Storage Pool               | The storage pool name the volume/storage class is associated with                                                                                  |
 | Storage System             | The storage system ID or IP address the volume is associated with                                                                                  |
-
+| Protocol                   | The storage system protocol type (nfs/iscsi) the volume/storage class is associated with                                                           |
```

Comment:
PowerStore and other arrays also supports FC.

Reply:
We can change the text, powerstore driver has only two creator